Modules/posixmodule.c

```diff
@@ -1373,7 +1373,6 @@ win32_get_reparse_tag(HANDLE reparse_point_handle, ULONG *reparse_tag)
 ** man environ(7).
 */
 #include <crt_externs.h>
-static char **environ;
```

Comment:
It is possible to simplify the rest of the code by doing:

`#define environ (*_NSGetEnviron())`

Then the new block at line 1398 can be dropped entirely.

Reply:
See resolved comments below. Victor prefers an explicit call to _NSGetEnvironment.

Reply:
Given the other `#ifdef` handling around there I agree with Victor.